interview-questions.md

```diff
@@ -16,40 +16,47 @@ INSTRUCTOR EXAMPLE: What is a conditional statement?
 1. What is git? What is the difference between git and Github?
 
   Your answer:
+  Git is software that allows programming collaboration amongst different parties remotely, whereas Github is a centralized location where code can be accessed by said parties. 
 
-  Researched answer:
-
+  Researched answer: 
+  Git is a version control system that helps programmers track changes in code and work together to complete a code base. GitHub is a hosting platform that allows for collaboration and version control. 
 
 
 2. Which JavaScript operators will return a Boolean value?
 
   Your answer:
+  The JavaScript operators that will return a Boolean value are operators that compare values and give a true or false result. These would be lesser than/greater than/equal to operators, == and === operators, and yes or no operators. 
 
   Researched answer:
-
+  As noted in our JavaScript Introduction material, the JavaScript operators that will return a Boolean value are logical operators and equality operators. 
```

Comment:
You mentioned in your answer less than and greater than operators which are known as relational operators and also return a boolean value.